modules/test/playwright/tests/frontend-editor-ckeditor-sample-web/stylesCombo.spec.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedLayoutTest} from '../../fixtures/isolatedLayoutTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export const test = mergeTests(isolatedLayoutTest(), loginTest());
+
+test('@LPD-33712 stylesCombo dropdown visible with maximized', async ({
+	layout,
+	page,
+}) => {
+	await test.step('Go to page and edit page', async () => {
+		await page.goto(layout.friendlyURL);
+
+		const editButton = page.getByRole('link', {name: 'Edit'});
+
+		await editButton.waitFor({state: 'visible'});
+		await editButton.click();
+	});
+
+	await test.step('Add sample ckeditor portlet on page ', async () => {
+		const searchFragment = page.getByLabel('Search Fragments and Widgets');
```

Comment:
probably we can reduce the amount of code here if we reuse some primitive to add a portlet to a page, pls find occurrences of `addPortlet`  in pw tests

Reply:
Hello @dsanz 
I made the changes! 
Let me know if I should change anything else.
Thank you!